frontend/src/components/common/edit-user-popup/edit-user-popup.tsx

```diff
@@ -40,7 +44,22 @@ const EditUserPopup: React.FC<Props> = ({ user, isShow, onEditUser, onFormHide }
     mode: 'onChange',
   });
 
-  const handleFormSubmit = (userData: IEditUserPayload) => onEditUser(userData);
+  const [image, setImage] = React.useState<string | undefined>(user.imagePath);
+
+  const handleUploadFile = (event: InputChangeEvent) => {
+    const DEFAULT_FILE_IDX = 0;
+    const file = event.target.files![DEFAULT_FILE_IDX];
```

Comment:
```suggestion
    const file = event.target.files[DEFAULT_FILE_IDX];
```
Rewrite it with `as`